.github/readme/synth.metadata/synth.metadata

```diff
@@ -4,14 +4,14 @@
       "git": {
         "name": ".",
         "remote": "https://github.com/googleapis/java-bigquerystorage.git",
-        "sha": "1554247cf55aa56281a530c721ab1650699a3efc"
+        "sha": "a6b53566cfd174fb36a903cbd84948defc0403e6"
```

Comment:
Maybe your rebase is not completely clean? Usually I don't see these files.

Reply:
Yeah I have no idea how this wound up like this. I will try and revert those files